### PR TITLE
edit test

### PR DESCRIPTION
**Pull request by BianChengNan**
bcn edit this test file,merge please

### CODE REVIEW

**Comment by xddd-iocp** [comment]
no I can't, sorry, by dd

**Comment by BianChengNan** [comment]
@xddd-iocp why you can't merge it

**Comment by xddd-iocp** [comment]
@BianChengNan no is no!!!!

**Comment by BianChengNan** [comment]
really neccessary!please review the code!

**Comment by BianChengNan** [comment]
@xddd-iocp Please!!!!

File: test [hunk 1]
```diff
@@ -1 +1,2 @@
+edit by bcn
 iiiiediti
```
Comment (BianChengNan) [inline test]: @xddd-iocp ok, this line seems rubbish

Comment (xddd-iocp) [inline test]: @BianChengNan  yes,it's rubbish. delete it please!

**Comment by BianChengNan** [comment]
@xddd-iocp now please merge!

**Pull request merged**